api/sip/client.py

```diff
@@ -587,11 +600,8 @@ def summary(self, hold_items=False, overdue_items=False,
             )
         return summary
     
-    def send(self, data, reset_sequence=False):
+    def send(self, data):
```

Comment:
reset_sequence was unused. I think it was left over from an earlier version of the code that resends a message if there's an error.